src/main/java/com/gmail/goosius/siegewar/SiegeController.java

```diff
@@ -111,8 +116,8 @@ public static boolean loadAll() {
 	public static void loadSiegeList() {
 		for (Town town : TownyUniverse.getInstance().getTowns())
 			if (SiegeMetaDataController.hasSiege(town)) {
-				System.out.println(SiegeWar.prefix + "Found siege in Town " + town.getName());
-				System.out.println(SiegeWar.prefix + "Loading siege of town " + town.getName());
+				System.out.println(SiegeWar.prefix + "Siege List Data: Found siege in Town " + town.getName());
+				System.out.println(SiegeWar.prefix + "Siege List Data: Loading siege of town " + town.getName());

```

Comment:
I'd drop this line, as we're really not loading the siege data yet.

Reply:
done